src/psyclone/nemo.py

```diff
@@ -393,8 +377,9 @@ class NemoLoop(Loop, NemoFparser2ASTProcessor):
     '''
     def __init__(self, ast, parent=None):
         from fparser.two.Fortran2003 import Loop_Control
+        valid_loop_types = Config.get().api_conf("nemo").get_valid_loop_types()
         Loop.__init__(self, parent=parent,
-                      valid_loop_types=VALID_LOOP_TYPES)
+                      valid_loop_types=valid_loop_types)
```

Comment:
I think this constructor expects valid_loop_types to be a list (https://psyclone-ref.readthedocs.io/en/latest/classpsyclone_1_1psyGen_1_1Loop.html) but we're passing it a dict here (I think this was my mistake originally). We probably get away with it but please make it explicit. Maybe by altering get_valid_loop_types() to return a list (if that's all we ever use it for) or doing `valid_loop_types=list(valid_loop_types)`.

Reply:
Yes, you are right. I've added a new function to the nemo config class that returns just a list.

Reply:
Thanks.